litestar/response/file.py

```diff
@@ -125,8 +125,14 @@ def __init__(
                 providing an :class:`os.stat_result`.
         """
         if not media_type:
-            mimetype, _ = guess_type(filename) if filename else (None, None)
+            mimetype, _encoding = guess_type(filename) if filename else (None, None)
```

Comment:
```suggestion
            mimetype, encoding = guess_type(filename) if filename else (None, None)
```
No need for this to be private.

Reply:
There's already an `encoding` variable in scope that eventually gets passed to the base `Response` constructor (it is used to populate the `content-type` header `charset`)

Reply:
Oh I see, I'd just name it something different then.